src/data/cassandra/Schema.hpp

```diff
@@ -953,6 +1004,15 @@ class Schema {
                 )
             );
         }();
+
+        // For ScyllaDB / Cassandra ONLY
+        std::optional<PreparedStatement> selectAccountFromBeginningScylla;
+        std::optional<PreparedStatement> selectAccountFromTokenScylla;
+        std::optional<PreparedStatement> selectNFTsByIssuerScylla;
+
+        // For AWS Keyspaces ONLY
+        // NOTE: AWS keyspace is not able to load cache with accounts
+        std::optional<PreparedStatement> selectNFTsAfterTaxonKeyspaces;
```

Comment:
Should be `std::variant<CassandraStatements, AWSKeyspacesStatements>`, and each one is a struct of needed statements, you will probably be able to get rid of std::optional as well